reference/twig_reference.rst

```diff
@@ -232,6 +232,18 @@ form_label
 Renders the label for the given field, more information in
 :ref:`the Twig Form reference <reference-forms-twig-label>`.
 
+form_help
+~~~~~~~~~~
+
+.. code-block:: twig
+
+    {{ form_help(view) }}
+
+``view``
+    **type**: ``FormView``
+
+Renders the text help for the given field.
```

Comment:
"Renders the text help" -> "Renders the help text" ?

Reply:
I've fixed it too, but apparently, don't recognized.